tests/JsonValidatorTest.php

```diff
@@ -183,28 +183,6 @@ public function testValidateFailsIfSchemaNeitherStringNorObject()
     /**
      * @expectedException \Webmozart\Json\InvalidSchemaException
      */
-    public function testValidateFailsIfSchemaFileContainsNoObject()
```

Comment:
Why did you remove these tests? How does the validate behave now in these cases?